README.md

```
@@ -1 +1,42 @@
+
+Welcome to the central `.github` repository for Brixion!
+
+This repository contains essential, organization-wide resources that help us
+maintain consistency, quality, and collaboration across all Brixion projects.
+
+Want to learn more about organization-wide repositories?
+[Read the GitHub documentation.](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/creating-a-default-community-health-file_)
+
+## 📦 What’s in this repository?
+
+#### 🐛 [Issue Templates](https://github.com/brixion/.github/tree/main/.github/ISSUE_TEMPLATE)  
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)